app/src/main/java/com/example/internetshop/presentation/viewModel/BottomNavViewModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.internetshop.presentation.viewModel
+
+import androidx.lifecycle.MutableLiveData
+import javax.inject.Inject
+
+class BottomNavViewModel @Inject constructor(): BaseViewModel() {
+    val bottomNavVisibility = MutableLiveData<Boolean>()
```

Comment:
Here is tautology `bottomNavViewModel.bottomNavVisibility`. This variable should be renamed to `visibility`. We understand that it's bottom nav visibility because we call this variable on bottomNavViewModel - `bottomNavViewModel.visibility`

Reply:
fixed